include/exadg/functions_and_boundary_conditions/interface_coupling.cpp

```diff
@@ -64,11 +70,67 @@ InterfaceCoupling<rank, dim, Number>::setup(
                                      dof_handler_src_.get_triangulation(),
                                      mapping_src_);
 
-    AssertThrow(
-      map_evaluator[quad_index].all_points_found() == true,
-      dealii::ExcMessage(
-        "Setup of InterfaceCoupling was not successful. Not all points have been found."));
+    if(not map_evaluator[quad_index].all_points_found())
+    {
+      std::string const file_name =
+        "interface_coupling_quad_index_" + dealii::Utilities::to_string(quad_index);
+      plot_points_and_triangulation(interface_data_dst->get_array_q_points(quad_index),
```

Comment:
Does one actually want to output all points and not only the few not found ones?

Reply:
https://www.dealii.org/developer/doxygen/deal.II/classUtilities_1_1MPI_1_1RemotePointEvaluation.html#af500b74c1b221de1168dabaabb885a6a